.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,53 @@
+name: Feature request
+about: Suggest an idea for this Paper
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for filling out a feature request for Paper! Please be as detailed as possible, so we may consider and review the request easier.
+        We ask that you search all the issues to avoid a duplicate feature request. If one exists, please reply if you have anything to add.
+        Before requesting a new feature, please make sure you are using the latest version and that the feature you are requesting is not already in Paper.
+
+  - type: dropdown
+    attributes:
+      label: Type of request
+      description: What type of request is this? Choose all that apply.
+      multiple: true
+      options:
+        - API Method
+        - Config option
+    validations:
+      required: true
```

Comment:
Will this work correctly? Neither of these could apply.

Reply:
This is a good point. 3 ways we can handle this:

1) Add an "Other" option (and more options in general)
2) Not require the "required" validation.
3) Remove this entirely (I'd just thought it'd be a bit more helpful)

Reply:
Generally, the one implementing the feature will take a stance to this. It might be best to just remove it.

Reply:
Removed in [`d914b2a` (#5278)](https://github.com/PaperMC/Paper/pull/5278/commits/d914b2a8513c71d05b264f26ea32d6192ac0361e)